spanner/tests/system/test_system.py

```diff
@@ -845,12 +845,11 @@ def test_read_w_index(self):
         temp_db = Config.INSTANCE.database(
             'test_read_w_index', ddl_statements=DDL_STATEMENTS + EXTRA_DDL,
             pool=pool)
-        operation = temp_db.create()
+        if not temp_db.exists():
```

Comment:
@chemelnucfin This won't help contention, because if it exists, then (at least) two tests would be trying to modify the same database.

You'd be better of just failing the test if `temp_db.exists()`. (Even this is not quite sufficient due to race conditions.)